extension.js

```diff
@@ -294,3 +294,135 @@ const getParamsFromGithubRaw = (url) => {
     const installedExtensions = getLocalStorageDataFromKey(LOCALSTORAGE_KEYS.installedExtensions, []);
     installedExtensions.forEach((extensionKey) => initializeExtension(extensionKey));
 })();
+const ITEMS_PER_REQUEST = 100;
+async function storeThemes() {
+    const BLACKLIST = await Blacklist();
+    let url = `https://api.github.com/search/repositories?q=${encodeURIComponent("topic:spicetify-themes")}&per_page=${ITEMS_PER_REQUEST}`;
+    const allRepos = await fetch(url).then(res => res.json()).catch(() => []);
+    if (!allRepos.items) {
+        Spicetify.showNotification("Too Many Requests, Cool Down.");
+    }
+    const filteredResults = {
+        ...allRepos,
+        items: allRepos.items.filter(item => !BLACKLIST.includes(item.html_url)),
+    };
+    return filteredResults;
+}
+async function storeExtensions() {
+    const BLACKLIST = await Blacklist();
+    let url = `https://api.github.com/search/repositories?q=${encodeURIComponent("topic:spicetify-extensions")}&per_page=${ITEMS_PER_REQUEST}`;
+    const allRepos = await fetch(url).then(res => res.json()).catch(() => []);
```

Comment:
This only fetches the first 100 repos returned. It should loop somehow until there are no results left. Something like how `loadPage()` works in `index.js` might work. Same goes for `storeThemes()`. 

```js
const currentPage = requestPage > -1 && requestPage ? requestPage : 1;
// Sets the amount of items that have thus been fetched
const soFarResults = ITEMS_PER_REQUEST * (currentPage - 1) + pageOfRepos.page_count;
const remainingResults = pageOfRepos.total_count - soFarResults;

// If still have more results, return next page number to fetch
console.log(`Parsed ${soFarResults}/${pageOfRepos.total_count} extensions`);
if (remainingResults > 0) return currentPage + 1; // There are more results. currentPage + 1 is the next page to fetch. 
else console.log("No more extension results");
```